Purchases/Public/RCPurchases.m

```diff
@@ -299,7 +299,7 @@ - (instancetype)initWithAppUserID:(nullable NSString *)appUserID
 
         [self.systemInfo isApplicationBackgroundedWithCompletion:^(BOOL isBackgrounded) {
             if (!isBackgrounded) {
-                [self.operationDispatcher dispatchOnWorkerThread:^{
+                [self.operationDispatcher dispatchOnWorkerThreadWithRandomDelay:NO :^{
```

Comment:
the value for `randomDelay` is always `isBackgrounded`, it might make sense to have the `operationDispatcher` depend on systemInfo and get the value without a param

Reply:
I actually think the param is better. That way we don't add logic to the operationDispatcher that depends on the state of the system. The operation dispatcher only knows what it means to add a random delay, it doesn't care if the app is backgrounded or not. Making `operationDispatcher` depend on the system info also means that you don't have flexibility to do the random delay only for getOfferings or getPurchaserInfo